internal/ingress/annotations/redirect/redirect.go

```diff
@@ -73,14 +73,25 @@ func (a redirect) Parse(ing *extensions.Ingress) (interface{}, error) {
 		return nil, err
 	}
 
+	const defaultPermanentRedirectCode = http.StatusPermanentRedirect
```

Comment:
The default should remain http.StatusMovedPermanently for backward compatibility reasons.

Reply:
I'd move this right above the definition of the Config struct, defaults should be clearly visible.